Cards/CardContainer.gd

```diff
@@ -393,3 +412,20 @@ func _update_card_positions() -> void:
 		# set position and rotation
 		movement_component.state_properties.desired_position = Vector2(card_x, card_y)
 		movement_component.state_properties.desired_rotation = rotation_amount
+
+func un_queue_card(card : CardWorld):
+	set_queued_card(null)
+	_unfocus_card(card)
+	card.get_card_movement_component().set_movement_state(Enums.CardMovementState.IN_HAND)
+
+func is_queued_card_in_play_area() -> bool:
+	return get_global_mouse_position().y < play_at_height 
+
+func _handle_queued_card():
+	if(queued_card == null):
+		return
+		
+	#check if right button is pressed
+	if(Input.is_action_just_pressed("cancel_queued_card_in_battle")):
```

Comment:
We're calling this in one spot, which is fine for now, but if any other file is going to utilize this action, we would want to create a global file to hold this in, so we can update it in one spot.

Reply:
I think it's a bit different because of the input map which already holds this

Reply:
I'm a bit confused, what is it that you want changed here?

Reply:
the fact you used a string "cancel_queued_card_in_battle" instead of using a variable that holds the string
but in fact, the string is linked to a variable (kinda) with the input map, meaning we already have a central place to hold changes to multiple instances if needed
the only problem we could have is if we decide we don't want to use the name "cancel_queued_card_in_battle", but technically that would be equivalent to changing the name of the variable in itself if we used one (so same problem).

At least if I understood correctly the input map

Reply:
So you think it's fine as is?

Reply:
I think this is fine for now, but if we see this being used in another file in another PR we should move it out to a global file.